src/utils/Logger.ts

```diff
@@ -273,11 +276,9 @@ class Logger {
     const writeLog = async (level: string, message: string) => {
       try {
         // If log folder not present create it
-        if (!(await RNFS.exists(this.getReactNativeLogsDir()))) {
```

Comment:
Does mkdir handle this check?

Reply:
Yep [mkdir](https://github.com/itinance/react-native-fs#mkdirfilepath-string-options-mkdiroptions-promisevoid) handles the check.